axelor-base/src/main/java/com/axelor/apps/base/service/administration/SequenceService.java

```diff
@@ -274,14 +274,19 @@ public String computeTestSeq(Sequence sequence, LocalDate refDate) {
     return computeNextSeq(sequenceVersion, sequence, refDate);
   }
 
-  protected String findNextLetterSequence(SequenceVersion sequenceVersion) {
+  protected String findNextLetterSequence(SequenceVersion sequenceVersion, Sequence sequence) {
```

Comment:
```suggestion
  protected String findNextLetterSequence(SequenceVersion sequenceVersion, int padding) {
```

After this change, please adapt the rest of the code